File: gluecodium/src/test/resources/smoke/defaults_const/output/android/com/example/smoke/StructWithPosEnums.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class StructWithPosEnums {
    public static final SomethingEnum FIRST_CONSTANT = SomethingEnum.REALLY_FIRST;
    @NonNull
    public SomethingEnum firstField;
    @NonNull
    public SomethingEnum explicitField;
    @NonNull
    public SomethingEnum lastField;
    public StructWithPosEnums() {
        this.firstField = SomethingEnum.REALLY_FIRST;
        this.explicitField = SomethingEnum.EXPLICIT;
        this.lastField = SomethingEnum.LAST;
    }
}

File: gluecodium/src/test/resources/smoke/defaults_const/output/android-kotlin/com/example/smoke/StructWithPosEnums.kt
/*

 *
 */

@file:JvmName("StructWithPosEnumsExtensions")

package com.example.smoke


class StructWithPosEnums {
    @JvmField var firstField: SomethingEnum
    @JvmField var explicitField: SomethingEnum
    @JvmField var lastField: SomethingEnum



    constructor() {
        this.firstField = SomethingEnum.REALLY_FIRST
        this.explicitField = SomethingEnum.EXPLICIT
        this.lastField = SomethingEnum.LAST
    }





    companion object {
        @JvmField final val FIRST_CONSTANT: SomethingEnum = SomethingEnum.REALLY_FIRST
    }
}

